extensions/wikia/Wall/WallBaseController.class.php

```diff
@@ -92,36 +101,34 @@ public function index( $wallMessagesPerPage = null ) {
 			$wallMessagesPerPage = self::DEFAULT_MESSAGES_PER_PAGE;
 		}
 
-		$this->getThreads( $title, $page, $wallMessagesPerPage );
+		/* @var Wall wall */
+		$wall = $this->getWallForIndexPage( $title );
+		$wall->setMaxPerPage( $wallMessagesPerPage );
+		$wall->setSorting( $this->getSortingSelected() );
+
+		$threads = $wall->getThreads( $page );
+		$countComments = $wall->getThreadCount();
+
+		$renderUserTalkArchiveAnchor = $this->request->getVal( 'dontRenderUserTalkArchiveAnchor', false ) != true;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "true" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1125)